Formula/adios2.rb

```diff
@@ -0,0 +1,73 @@
+class Adios2 < Formula
+  desc "Next generation of ADIOS developed in the Exascale Computing Program"
+  homepage "https://adios2.readthedocs.io"
+  url "https://github.com/ornladios/ADIOS2/archive/v2.5.0.tar.gz"
+  sha256 "7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329"
+  head "https://github.com/ornladios/ADIOS2.git", :branch => "master"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "c-blosc"
+  depends_on "libfabric"
+  depends_on "libpng"
+  depends_on "mpi4py"
+  depends_on "numpy"
+  depends_on "open-mpi"
+  depends_on "python"
+  depends_on "zeromq"
+  uses_from_macos "bzip2"
+
+  patch do
```

Comment:
@ax3l you rock! This is exactly what we need here.

The only thing that it's missing is a comment indicating when to remove the path, and a very very brief description of the problem. I presume it should be removed in the next major release. Could you please add a comment providing these two details?